notebooks/0.download-data/0.download-data.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# activate conda env
+conda activate cfret-map
+
+# convert notebooks into python scripts
+jupyter nbconvert --to python --output-dir=nbconverted/ *.ipynb
```

Comment:
Consider using a tool which allows you to convert the notebooks into code automatically instead of having to run a script (for example, [jupytext](https://jupytext.readthedocs.io/en/latest/)). This would provide benefits with concerns to development (no guessing which script must be used to convert files or when to run it), remove the need for this and other files like it, and increase your development speed (reduce your steps to reach a PR).

If you want to stick with this file and files like it for conversion, consider using a [Makefile](https://book.the-turing-way.org/reproducible-research/make/make-examples.html), [justfile](https://github.com/casey/just), or [poethepoet](https://poethepoet.natn.io/index.html) to centralize how and when development tasks are run.

Reply:
This topic extends beyond the current scope and leans more toward a software setup issue. I’ll create a dedicated issue to focus on configuring these packages as part of a future feature. These tools are really neat!

issue created #10